Source/p_mobj.c

```diff
@@ -769,6 +797,9 @@ mobj_t *P_SpawnMobj(fixed_t x, fixed_t y, fixed_t z, mobjtype_t type)
   mobj->thinker.function = P_MobjThinker;
   mobj->above_thing = mobj->below_thing = 0;           // phares
 
+  //e6y
```

Comment:
Again, let's please not adapt bad practice of using three-letter comments that don't say anything about which code was changed and why.